### ISSUE TITLE
Flesh out Newbridge Site and hand over to them

### ISSUE THREAD

**Issue by ColmMassey** [SolidarityEconomyAssociation/sea-wordpress-theme#3]
If
https://github.com/SolidarityEconomyAssociation/sea-wordpress-theme/issues/2

shows that it isn't complete, we may need some more dev before we can complete this it.

**Comment by ColmMassey** [comment]
The Zapdev site is now pretty close to the target website now. There are a few small issues. Next we need to have another attempt at doing this on wp-0.

**Comment by ColmMassey** [comment]
Also should list here what is still missing on Zapdev.

**Comment by wu-lee** [comment]
There's been a bit of discussion about what URL to give the Wordpress hosts, if the maps are on `https://<topic>.solidarityeconomy.coop`.  I did suggest putting them all on the same domain, under different paths, and emailed Web Architects about it.

However, I think the simplest thing might be to turn it around: put all the maps on the same domain, under different paths, and the WP instances on their own domains.

So for Newbridge, point https://newbridge.solidarityeconomy.coop/ at the Wordpress instance, and move the map to (say) https://maps.solidarityeconomy.coop/newbridge/. After all, the map URL is the one we don't really care about because people won't see it. 

Any comments @ColmMassey ?

**Comment by ColmMassey** [comment]
Happy with that I think. And so for the maps that have no mini-website, like dotcoop, would they stay as they are? or would we need to change them to redirect to the new map location? Would that effect certificates?

**Comment by wu-lee** [comment]
> And so for the maps that have no mini-website, like dotcoop, would they stay as they are? or would we need to change them to redirect to the new map location? Would that effect certificates?

As we just discussed on a call, they could stay as they are.  Certificates: there might need to be one more if we make a new domain for maps to share, but otherwise also can stay the same.

**Comment by wu-lee** [comment]
Just to update here:
- There's now a WP instance for this hosted with Web Architects, originally on https://wp.solidarityeconomy.coop
- I've installed sea-wordress-theme on it and imported the content from https://sea-test.dreamhosters.com/
- This was referring to the map at https://newbridge.solidarityeconomy.coop/map.html
- This seemed to be working ok, except for a few things which didn't get set via the content import, like the site title, fonts, title image, etc.
- Therefore I've asked Web Architects to update the domain newbridge.solidarityeconomy.coop to point to the WP server, and reconfigure the WP install to work on this
- I think this has been done now, works from some places, although the DNS changes don't seem to have propagated all the way to my desktop
- I've installed the development version of the newbridge map here: https://dev.maps.solidarityeconomy.coop/newbridge/

So when the DNS details straighten themselves out this should be all working with WP on  https://newbridge.solidarityeconomy.coop/ embedding the map on https://dev.maps.solidarityeconomy.coop/newbridge/

@ColmMassey What's needed now is for you check it over, and make sure there no missing pieces, and the content, fonts, etc. all look adequate from your point of view.

**Comment by ColmMassey** [comment]
As newbridge.solidarityeconomy.coop is the production version it shouldn't point to the dev version of the map, but the working, older one.

**Comment by ColmMassey** [comment]
> I've installed the development version of the newbridge map here: https://dev.maps.solidarityeconomy.coop/newbridge/

Shouldn't this just be the sea-map instance and not the mini-website?

**Comment by ColmMassey** [comment]
>What's needed now is for you check it over, and make sure there no missing pieces, and the content, fonts, etc. all look adequate from your point of view.

Do I have admin privilidges for this site yet?

**Comment by wu-lee** [comment]
> it shouldn't point to the dev version of the map, but the working, older one.

I've updated it to point to the `master` branch of `newbridge-project` deployed on https://maps.solidarityeconomy.coop/newbridge/map.html

> Shouldn't this just be the sea-map instance and not the mini-website?

Yes - but the WP widget points to the embedded map, not the mini-website. The mini-website is still there, just not visible on WP.  The reason I've not removed the mini-website is just that this is the simplest thing to do at this point - I'll add a ticket to remove it.

> Do I have admin privilidges for this site yet?

There is an account for you imported with all the content. However, I see it's not been given the admin role, but I've corrected that now. I've sent a password reset just in case the password isn't set correctly.
